node_modules/oae-authentication/lib/api.js

```diff
@@ -87,7 +87,6 @@ var _configUpdate = function(tenantAlias) {
     }
 };
 
-ConfigAPI.on('cleared', _configUpdate);
```

Comment:
Can we do this? I can't remember of the top of my head but are there some strategies that we configure globally?